### PR TITLE
New URDF/XACRO files and URDF import tutorial

### PR DESCRIPTION
**Pull request by pawelbudziszewski**

### CODE REVIEW

File: Project/Assets/robotec_freezed_urdf_chasis/kraken_chasis.old.xacro [hunk 1]
```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" ?>
```
Comment (michalpelka) [inline Project/Assets/robotec_freezed_urdf_chasis/kraken_chasis.old.xacro:1]: I would recommend to clean unused files

**Comment by adamdbrw** [comment]
@michalpelka Is this going to be heavily affected by your changes to URDF Import?
This is also demo-specific. I wonder if we could make such a tutorial Gem-side with a some example URDF in the Robot Importer (having it as a sub-Gem).

**Comment by michalpelka** [comment]
@adamdbrw It won't be affected by URDF import changes.
The changes in branch https://github.com/RobotecAI/o3de-ros2-gem/pull/297 are refactoring / with some extra features.
One possibility is that we would probably run XACRO inside import context, so URDF files can be safely removed then.

**Comment by adamdbrw** [comment]
@pawelbudziszewski @michalpelka I would like to review and merge this PR. Does it need updates regarding models, perhaps some unwanted files? Any remaining action items before review?

**Comment by michalpelka** [comment]
Please remove unused models e.g.: 282
[Project/Assets/robotec_freezed_urdf_chasis/kraken_chasis.old.xacro](https://github.com/o3de/ROSConDemo/pull/173/files#diff-da122835ea4e7bc2c8ca43d3be360605a39761f69fe91007837af4c46d1f44e4) - ideally let us keep only one XACRO. Do not attach processed URDF.

**Comment by adamdbrw** [comment]
Updated for:
- clarity
- wording, more concise phrasing, removing repetitions. 
- context (why are we doing these steps?)
- URDF reference
- link from README (so people actually can reach this)

**Review by adamdbrw — APPROVED** [review]

**Pull request merged**